misc/multistream-select/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # 0.10.4 [unreleased]
 
+- Make default features of `libp2p-core` optional.
+  [PR 2181](https://github.com/libp2p/rust-libp2p/pull/2181)
+
```

Comment:
```suggestion
```

No need for this entry, given that `libp2p-core` is only a dev-dependency to `multistream-select`.